.github/workflows/figma_connect_publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Figma Connect Publish on Change
+
+on:
+  push:
+    paths:
+      - packages/react/src/components/**/*.figma.tsx
+    branches:
+      - main
+
+jobs:
+  code-connect:
+    name: Code Connect
+    runs-on: ubuntu-latest
+    steps:
+      - run: npx figma connect publish
+        env:
+          FIGMA_ACCESS_TOKEN: ${{ secrets.FIGMA_ACCESS_TOKEN }}
```

Comment:
Just wanted to ask, how does this access token get generated?

Reply:
I have to manually create it in Figma and than it has to be added to the repo. Sadly there is no org token available in Figma yet. The Figma folks are aware of this issue though.